pages/ag/dmv-rse.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: DMV-RSE Affinity Group
+subtitle:
+menubar:
+permalink: /ag/dmv-rse
+set_last_modified: true
+---
+
+# Overview
+
+DMV (Delaware-Maryland-Virginia) RSE is a regional affinity group for professionals in RSE and RSE-adjacent disciplines. We organize informal gatherings where interested professionals can connect and learn from each other.
+
+# Membership
+
+DMV-RSE is open to anyone who considers themselves a research software engineer and lives in the DMV area, regardless of job title or domain. This includes workers in academic research (students, postdocs, and professors), as well as RSE professionals in government, industry and other sectors (e.g. non-profit orgs).  
```

Comment:
Suggestion: "who considers themselves a research software engineer or ally" - unless you do want to specifically restrict to RSEs?

Reply:
Thanks @mrmundt ! Yeah, that sounds good -- I added it in.